proto/osmosis/lockup/tx.proto

```diff
@@ -88,4 +89,16 @@ message MsgForceUnlock {
   ];
 }
 
-message MsgForceUnlockResponse { bool success = 1; }
\ No newline at end of file
+message MsgForceUnlockResponse { bool success = 1; }
+
+message MsgRebondTokens {
+  string owner = 1 [ (gogoproto.moretags) = "yaml:\"owner\"" ];
+  uint64 ID = 2;
+  // Amount of tokens to rebond. Rebond all if not set.
+  repeated cosmos.base.v1beta1.Coin coins = 3 [
```

Comment:
other messages in `lockup` to which some amount of coins should be provided has this field named `coins`, I think if we change it here, it would be nice to change it in other msgs, wdyt?

Reply:
I agree that coins is a better name here, I think `rebondingMsg.coins` is very clear. Also my (slightly pedantic) take is that those coins aren't "rebonding" as they are just part of the request.